cmd/skywire-cli/commands/hv/root.go

```diff
@@ -17,6 +17,8 @@ var (
 	pk      string
 	url     string
 	pkg     bool
+	port    int
+	ipadd   string
```

Comment:
Please try to use somewhat canonical names at least in your own part of the codebase. So something like `ip` or `ipAddr`.

Reply:
changed